### ISSUE TITLE
Question: Is it possible to upload your gcode file?

### ISSUE THREAD

**Issue by mptrs** [bjoernkarmann/project_alias#11]
Hi,

Is it possible for you to upload the gcode file. I'm would like to print it with the same settings that worked for you.

Thanks,

Michiel

**Comment by moritzgloeckl** [comment]
I would not recommend you printing directly from a gcode file unless you have the exact same 3D printer. Why not slice it yourself?

**Comment by mptrs** [comment]
I think it's possible we have the same printer :) Prusa i3 (mk2.5)
I did try to slice it myself, but I'm having trouble with the orientation.

**Comment by bjoernkarmann** [comment]
As mentioned in the thread, I don't think it is a good idea to use the same g-code. But I can share the orientation and settings I used for my prints. Attached is a screengrab from slic3r where the back face has been selected. Next, I added support materials from build plate only and tried to keep the overhang threshold as low as possible (I think I had mine set to 30 or 35 deg). The layer height was set to 0.15mm together with a decent brim of 18mm.

<img width="556" alt="shell-orientation" src="https://user-images.githubusercontent.com/14107560/51499235-bb543200-1dc9-11e9-87b9-63f429954ed1.png">
